### ISSUE TITLE
Include `digestMultibase` in data integrity context

### ISSUE THREAD

**Issue by PatStLouis** [w3c/vc-data-integrity#317]
The [resource integrity section](https://www.w3.org/TR/vc-data-integrity/#resource-integrity) refers to the `digestMultibase` property and its usage, however this property is not present in the data integrity context. It is, however, present in the vcdm 2.0 context, as described by the following text:
> JSON-LD context authors are expected to add digestMultibase to contexts that will be used in documents that refer to other resources and to include an associated cryptographic digest. For example, the [Verifiable Credentials Data Model v2.0](https://www.w3.org/TR/vc-data-model-2.0/) base context (https://www.w3.org/ns/credentials/v2) includes the digestMultibase property.

I want to suggest adding the `digestMultibase` to the data integrity context, keeping the same definition as is in the vcdm 2.0 context.
```
"digestMultibase": {
  "@id": "https://w3id.org/security#digestMultibase",
  "@type": "https://w3id.org/security#multibase"
}
```

Any particular reason it was not included?

**Comment by dlongley** [comment]
This was discussed in #241 with a resolution via #264. I recommend we close this as a duplicate; please see the discussion on #241.

**Comment by msporny** [comment]
This doesn't seem to be a duplicate... having re-read the issue.

Looks like it was added to the v1 context but then became missing from the v2 context.

**Comment by dlongley** [comment]
> This doesn't seem to be a duplicate... having re-read the issue.
> 
> Looks like it was added to the v1 context but then became missing from the v2 context.

I think what happened here is that an explanation and consensus for why we wouldn't be adding it to any DI contexts happened in #241. We had approvals to proceed with the related PR (#264) which, at the time of the approvals, only added text and didn't change the DI contexts. However, then it looks like a last minute attempt by the editor to try and fix up the PR before merging it accidentally added a commit that added the term to the DI v1 context.

We could leave things as they are in the DI v1 context, which is a context that shouldn't be used anyway as it isn't in sync with the DI specs (that's why we have DI v2). However, I think the best course of action is to revert that additional accidental commit so that people don't erroneously use the DI v1 context (e.g., to make use of that term) and then find themselves getting into interop trouble. Reverting that commit will also fix any problems in the community with software that had already considered the DI v1 context immutable (before DI v2 came out) and won't have seen that change anyway. All contexts are non-normative and this shouldn't be otherwise disruptive.

So I recommend we close this issue and revert that one commit that came in after the approvals on #264 that changed the DI v1 context.

**Comment by msporny** [comment]
> So I recommend we close this issue and revert that one commit that came in after the approvals on #264 that changed the DI v1 context.

Yep, agreed with your analysis @dlongley. Looks like I pushed an outdated commit that shouldn't have been pushed right before the merge, which is what led us to this point. Agree that we should remove `digestMultibase` from the v1 context as well to make sure people don't end up at the same conclusion that @PatStLouis did (which was a logical place to land based on the issues/PRs/commits that he had seen).

I'll do a few commits to put everything back to the way it was intended to be and link to them in this issue.

**Comment by msporny** [comment]
Fixed in 2b2788232d28d201773dda3b43e0127db9fbf96a. Closing.
